internal/templates/common/yamls/clusterIssuer.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if $.Values.app.secret }}
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: {{ $.Values.app.secret }}-clusterissuer
+spec:
+  ca:
+    secretName: {{ $.Values.app.secret }}
```

Comment:
When referencing a Secret resource in ClusterIssuer resources, the Secret needs to be in the same namespace as the cert-manager controller pod. Do we have some checks for it already?

We can optionally override this by using the --cluster-resource-namespace argument to the controller but not sure if we are going to do it.

Reply:
Good call. I added a check for the secret (in ns `cert-manager`) to the reconciler.